src/drivers/amd/common/acp_dmic_dai.c

```diff
@@ -18,6 +18,10 @@
 #include <sof/lib/io.h>
 #include <platform/chip_offset_byte.h>
 
+extern uint32_t num_chs;
```

Comment:
procedural NAK on the commit message: please make sure the patch title has a prefix that identifies the vendor and subsystem, e.g. 

AMD: DMIC: Introduce initial silence in DMIC path to remove the popping noise

Reply:
changed commit message